keras_cv/layers/object_detection/multi_class_non_max_suppression.py

```diff
@@ -71,6 +72,12 @@ def call(
                 `bounding_box_format` specified in the constructor.
             class_prediction: Dense Tensor of shape [batch, boxes, num_classes].
         """
+        if multi_backend() and keras.backend.backend() != "tensorflow":
```

Comment:
This is some fancy NMS that takes the predicted class into account?

Reply:
Indeed -- and we'll be able to support it in Torch as well (coming soon)